config/sidekiq_scheduler.yml

```diff
@@ -194,3 +194,8 @@ CypressViewportUpdater::UpdateCypressViewportsJob:
   cron: "0 12 3 * * America/New_York"
   class: CypressViewportUpdater::UpdateCypressViewportsJob
   description: "Updates Cypress files in vets-website with data from Google Analytics."
+
+ClaimsApi::ClaimAuditor:
+  cron: "0 0 0/12 1/1 * ? * America/New_York"
```

Comment:
So this job runs "Every 12 days every 1 months at 12:00am"?

Reply:
Hmmm, isn't this "_every 12hrs of every day_"?  🤔 

```
1. | 2021-03-09 Tue 12:00:00
2. | 2021-03-10 Wed 00:00:00
3. | 2021-03-10 Wed 12:00:00
4. | 2021-03-11 Thu 00:00:00
5. | 2021-03-11 Thu 12:00:00
```

Reply:
yeah, it should be every 12 hours essentially